openbb_terminal/stocks/insider/sdk_helper.py

```diff
@@ -0,0 +1,229 @@
+"""Insider SDK Helpers"""
+__docformat__ = "numpy"
+
+import pandas as pd
+
+from openbb_terminal.stocks.insider import openinsider_model
+
+
+def stats(symbol: str) -> pd.DataFrame:
```

Comment:
Feels like `stats` and `insider_filter` could be 1 function only, but probably it's easier to use it like this from the user perspective.

Reply:
Matching to terminal functions again.  They also do different things.  One requires a filter, one requires a ticker